Gordon360/Models/CCT/Activity.cs

```diff
@@ -0,0 +1,58 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT
+{
+    [Table("Activity", Schema = "RecIM")]
+    public partial class Activity
+    {
+        public Activity()
+        {
+            ParticipantActivity = new HashSet<ParticipantActivity>();
+            Series = new HashSet<Series>();
+            Team = new HashSet<Team>();
+        }
+
+        [Key]
+        public int ID { get; set; }
+        [Required]
+        [StringLength(128)]
+        [Unicode(false)]
+        public string Name { get; set; }
+        [StringLength(128)]
+        [Unicode(false)]
+        public string Logo { get; set; }
```

Comment:
TODO last, otherwise DB changes will break frontend work.

Reply:
@bennettforkner I actually don't know if it's a path to the server or a logo URL. Our image implementation is completely missing because we're disconnected from any image fetching. (Also I'm pretty sure it's been on the todo list for a couple months now and no one has touched it)